services/s3/src/main/java/software/amazon/awssdk/services/s3/multipart/MultipartDownloadType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.s3.multipart;
+
+/**
+ *
+ * todo complete when implementing multipart download
+ */
+public enum MultipartDownloadType {
+    PART,
+    RANGE;
+}
```

Comment:
What are the pros/cons for letting the customer pick this instead of just having us pick this for them? Permissions makes sense as one thing.